modules/standard/IO.chpl

```diff
@@ -1680,9 +1682,10 @@ operator file.=(ref ret:file, x:file) {
   ret._file_internal = x._file_internal;
 }
 
-private proc initHelper(ref f: file, fp: c_FILE, hints=ioHintSet.empty,
+private proc initHelper(ref f: file, fp, hints=ioHintSet.empty,
```

Comment:
Can you add another todo comment here to put back the type annotation post deprecation?